worker/src/main/scala/org/apache/celeborn/service/deploy/worker/WorkerSource.scala

```diff
@@ -111,8 +111,12 @@ class WorkerSource(conf: CelebornConf) extends AbstractSource(conf, Role.WORKER)
     val applicationIds = appActiveConnections.get(client.getChannel.id().asLongText())
```

Comment:
could you define a variable for `metricsLabelApplicationId`, and how about return directly if `metricsLabelApplicationId` is false?